webauthn4j-core/src/test/java/com/webauthn4j/validator/BeanAssertUtilTest.java

```diff
@@ -143,8 +143,9 @@ void validate_WebAuthnRegistrationContext_test() {
 
```

Comment:
IMO, it seems to be better to write tests for the newly introduced methods :
``` java
validate(WebAuthnRegistrationData)
validate(WebAuthnRegistrationParameters)
validate(WebAuthnAuthenticationData)
validate(WebAuthnAuthenticationParameters)
```

Reply:
Thank you for pointing out. I add test cases in https://github.com/webauthn4j/webauthn4j/pull/183/commits/cd318f6053d389b2328d79369d842db4a02b23bd